plugins/woocommerce/includes/wc-update-functions.php

```diff
@@ -2558,3 +2558,34 @@ function wc_update_722_adjust_new_zealand_states() {
 function wc_update_722_adjust_ukraine_states() {
 	return wc_update_721_adjust_ukraine_states();
 }
+
+/**
+ * Add new columns date_paid and date_completed to wp_wc_order_stats table in order to provide the option
+ * of using the dates in the reports
+ */
+function wc_update_740_add_columns_to_order_stats_table() {
+	global $wpdb;
+
+	$wpdb->get_var(
+		"ALTER TABLE wp_wc_order_stats
+		ADD COLUMN date_paid DATETIME DEFAULT '0000-00-00 00:00:00',
+		ADD COLUMN date_completed DATETIME DEFAULT '0000-00-00 00:00:00';"
+	);
+
+	$wpdb->get_var(
```

Comment:
I think using `$wpdb->query` might be slightly more semantically correct, though they will accomplish the same thing.

Reply:
I'm also not sure if we should handle this and the following update here.  In the past, we've often relied on reimporting report data as the way to make sure data is up-to-date.  Though this single query might be more performant than needing to re-import all data.

Not overly opinionated on this either way, just leaving some thoughts.